src/freertos_drivers/common/TCAN4550Can.hxx

```diff
@@ -55,8 +57,10 @@ public:
     /// @param name name of this device instance in the file system
     /// @param interrupt_enable callback to enable the interrupt
     /// @param interrupt_disable callback to disable the interrupt
+    /// @param test_pin test GPIO pin for instrumenting the code
     TCAN4550Can(const char *name,
-                void (*interrupt_enable)(), void (*interrupt_disable)())
+                void (*interrupt_enable)(), void (*interrupt_disable)(),
+                const Gpio *test_pin = DummyPinWithRead::instance())
```

Comment:
do you want to ifdef this argument as well? (would avoid a reference to DummyPinWithRead::instance(), which is code in flash, vtable, etc.)

Reply:
Fixed.